helpers/dbload.ts

```diff
@@ -66,6 +66,14 @@ User.belongsToMany(Lesson, {
   through: { model: UserLesson }
 })
 
+User.hasMany(Submission, {
+  foreignKey: 'userId'
+})
+
+User.hasMany(UserLesson, {
```

Comment:
This is incorrect. UserLesson is a join table that is used to manage many to many relationships

Reply:
We can resolve this.

Reference 
https://sequelize.org/master/manual/advanced-many-to-many.html